roles/lnls-ans-role-sirius-hlacons/README.md

```diff
@@ -0,0 +1,86 @@
+LNLS Ansible Role Sirius HLA Apps
+=================================
+
+This Ansible role configures some defaults Sirius High Level Applications for Sirius Light Source control machines.
+
+## Requirements
+
+- ansible >= 2.4
+- molecule >= 2.20
+
+## Role Variables
+
+
+## Example Playbook
+
+```yaml
+---
+- hosts: all
+  tasks:
+  - import_role:
+      name: '{{playbook_dir}}'
```

Comment:
Ansible lint complain about the syntax '{{playbook_dir}}'. We should use spaces between {{, as in
"{{ playbook_dir }}